documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst

```diff
@@ -1146,63 +1025,47 @@ where:
 
    Like ``line_proj``, ``cylinder_proj`` can be :ref:`defined using
    positional or keyword arguments <poskwarg>`. However the same
-   recommendation applies that positionals should only be used to
+   recommendation applies that positional should only be used to
    define ``u`` and ``v``.
+   
+``cylinder_proj`` defines a cylindrical coordinate system with
+cylindrical coordinates of momentum transfer vector  :math:`\vec{Q}`.
+Energy transfer coordinate for ``cylinder_proj`` remain unchanged. 
+If projection ``offset`` parameter is zero, this vector is the vector
+of momentum transfer from neutron to excitations -- lattice measured
+in scattering experiment. If offset is non-zero, :math:`\vec{Q}`
+is the difference between ``offset`` and the measured momentum transfer.
+
+Similarly to :ref:`Spherical projections <Spherical_Projections>`, we introduce auxiliary 
+orthogonal coordinate system, which unit vector :math:`\vec{e_z}` being parallel to 
+:math:`\vec{u}` and unit vector :math:`\vec{e_x}` is orthogonal to :math:`\vec{e_z}`
+and lies in the plane defined by :math:`\vec{u}` - :math:`\vec{v}`. Cylindrical 
+coordinates of the vectors of interest are calculated in this system. 
+(see :ref:`Cylinder coordinates <fig_cylinder_coodinates>` below.)
+When crystal lattice is orthogonal, vectors :math:`\vec{e_z}` is aligned with :math:`\vec{u}` and 
+vector :math:`\vec{e_x}` is aligned with :math:`\vec{v}`.
 
 ``cylinder_proj`` defines a cylindrical coordinate system, where:
 
-* :math:`q_{\|}`
-
-  The length along the polar axis :math:`\vec{u}` to :math:`\vec{q}` relative to
-  the origin (``offset``) in :math:`hkl`
-
-  Mathematically, this is defined as:
-
-  .. math::
-
-     \vec{q_{\|}} = \frac{\left(\vec{q} \cdot{} \vec{u}\right)}
-                         {\left|u\right|}
-                    \vec{u}
-
-  .. math::
-
-     q_{\|} = \left| \vec{q_{\|}} \right|
-
-* :math:`q_{\perp}`
-
-  The radial distance from the polar axis :math:`\vec{u}` to :math:`\vec{q}`
-  relative to the origin (``offset``) in :math:`hkl`.
-
-  Mathematically, this is defined as:
+- :math:`\vec{Q_\perp}=\vec{Q}-\vec{e_z}(\vec{e_z}\cdot \vec{Q})`  --
+  The length of the orthogonal to axis :math:`\vec{e_z}` part of the momentum transfer 
+  :math:`\vec{Q}` measured from the ``cylinder_proj``  origin (``offset``) in :math:`hkl`.
```

Comment:
Again, formulas removed, simplified descrition and reference to basis coordinate system have been added